src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -224,6 +224,11 @@ public enum NuGetLogCode
         /// </summary>
         NU3012 = 3012,
 
+        /// <summary>
+        /// Certifiate has a CNG private key
```

Comment:
📝 Typo on certificate.
📝 This description looks like the opposite of what the actual problem is. I would expect something closer to the HRESULT description.